calculators/items/Recombobulator_3000.js

```diff
@@ -1,17 +1,10 @@
-const { APPLICATION_WORTH } = require('../../constants/applicationWorth');
 const { ALLOWED_RECOMBOBULATED_CATEGORIES, ALLOWED_RECOMBOBULATED_IDS } = require('../../constants/misc');
+const { APPLICATION_WORTH } = require('../../constants/applicationWorth');
+const { ItemHandler } = require('../handlers/ItemHandler');
 
-class Recombobulator3000Calculation {
-    constructor({ calculation, itemData, prices, itemId, price, itemLore, isRecombobulated, skyblockItem }) {
-        this.isRecombobulated = isRecombobulated;
-        this.skyblockItem = skyblockItem;
-        this.calculation = calculation;
-        this.itemLore = itemLore;
-        this.itemData = itemData;
-        this.prices = prices;
-        this.itemId = itemId;
-        this.price = price;
-        // this.calculate();
+class Recombobulator3000Calculation extends ItemHandler {
+    constructor() {
+        super();
```

Comment:
No need to extend if you just call super()? Also why do we have data in the handlers? Why not static class methods that take in the item and prices?

Reply:
Altpapier wanted it so we don't have to pass in any parameters in Handlers, so it just extends main handler class which has all parameters. Mostly just to keep the code cleaner I guess, cuz it's a lot of copy pasting, but the way you've done is good as well, passing in only item & prices